src/components/Card/CategoryCard/CategoryCard.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+function CategoryCard({category,img}) {
```

Comment:
you are blocking component receiving additional props via rest

Reply:
Thanks for letting me know. Added rest.